### PR TITLE
fix: add regV2 metadata in vaults service calls

### PR DESCRIPTION
**Pull request by NamelsKing**

### CODE REVIEW

File: src/registry/interfaces/registry-vault.interface.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { VaultState } from '../../api';
+import { VaultRegistryV2Entry } from '../../registry.v2/interfaces';
```
Comment (axejintao) [inline src/registry/interfaces/registry-vault.interface.ts:2]: can we add this to the registry interfaces export index.ts

**Review by axejintao — APPROVED** [review]

File: src/registry/interfaces/registry-vault.interface.ts (hunk 1 above)
Comment (NamelsKing) [reply src/registry/interfaces/registry-vault.interface.ts:2]: ofc

**Pull request merged**